src/modules/DressMe/PersonalDeatails/FieldAndValidation.tsx

```diff
@@ -0,0 +1,51 @@
+import { Field, useField } from "formik";
+import styled from "styled-components";
+
+export interface FieldProps {
+  placeholder: string;
+  name: string;
+  type?: string;
+  min?: string;
+  max?: string;
+  step?: string;
+}
+
+const FieldAndValidation = ({ min, max, step, type, placeholder, name }: FieldProps) => {
+  const [Field, Meta] = useField(name);
+  return (
+    <FieldContainer>
+      <PersonalField
+        min={min}
+        max={max}
+        step={step}
+        type={type}
+        placeholder={placeholder}
+        name={name}
+      />
+      {Meta.error ? <Invalid>{Meta.error}</Invalid> : null}
```

Comment:
check Formik documentation for error handling, maybe they have a designated api/component for that